sprint0/sprint0/Weapons/Boomerang.cs

```diff
@@ -85,6 +85,10 @@ public void Update()
                 currFrame = (currFrame + 1) % (totalFrames * repeatedFrames);
                 age++;
             }
+            if (Shooter is Goriya goriya1)
+                alive = goriya1.health > 0;
```

Comment:
I do not think that should be an issue because this is handling the edge cases where there is a flying boomerang being thrown by a no longer alive goriya. So we wouldn't want that specific instance of a boomerang to come alive because the goriya isn't coming back alive but also future boomerangs that will be created with a reset alive value.